templates/deploy-ansible-service-broker.template.yaml

```diff
@@ -112,14 +112,14 @@ objects:
   metadata:
     name: access-asb-role
   rules:
-  - nonResourceURLs: ["${BROKER_URL_PREFIX}", "${BROKER_URL_PREFIX}/*"]
+  - nonResourceURLs: ["/${BROKER_URL_PREFIX}", "/${BROKER_URL_PREFIX}/*"]
```

Comment:
What's the necessity for this change?

Reply:
I needed to make changes in other areas in the template as well to include the '/'.  It's the way to make consistent with how the 'run_latest.build.sh' script specifies the NAMESPACE parameter (without '/').

Reply:
I will update